### PR TITLE
media-converter/videoconv.rs: help with FIXME and use of more functional for

### PR DESCRIPTION
**Pull request by orsenkucher**
Hi, I was traveling across the rusty part of Proton
and noticed those FIXME comments in `videoconv.rs`, which I could totally help with
So, here it is😄

### CODE REVIEW

File: media-converter/src/videoconv.rs [hunk 1]
```diff
@@ -216,10 +216,7 @@ impl<'a> Read for PadReader<'a> {
         if to_copy == out.len() {
             out.copy_from_slice(&self.chunk[self.chunk_offs..(self.chunk_offs + to_copy)]);
         }else{
-            /* FIXME: there's probably some cool functional way to do this */
-            for i in 0..to_copy {
-                out[i] = self.chunk[self.chunk_offs + i];
-            }
+            (0..to_copy).for_each(|i| out[i] = self.chunk[self.chunk_offs + i]);
```
Comment (darksv) [inline media-converter/src/videoconv.rs]: I think the whole `if` can be replaced with just `out[..to_copy].copy_from_slice(&self.chunk[self.chunk_offs..(self.chunk_offs + to_copy)]);` 😄

Comment (orsenkucher) [reply media-converter/src/videoconv.rs]: done👍

**Comment by aeikum** [comment]
@orsenkucher @darksv Thanks! I pushed one more commit to your branch that fixes up the other chunk to avoid the if-statement as well. Does it look good?

**Comment by orsenkucher** [comment]
@aeikum Yup👍 looks good to me

**Comment by aeikum** [comment]
This was merged as 370dc01d07eba692e6d19ceebc7b9e0da07ad0c2. Thanks!

**Pull request closed without merging**